apps/docs/stories/components/payment-form.stories.tsx

```diff
@@ -133,16 +113,22 @@ const meta: Meta = {
 }
 
 const Template = (args: any) => {
+  // Set the prop only if it is defined
+  const getAttributeString = (attrName: string) => {
+    return args[attrName] ? `${attrName}="${args[attrName]}"` : '';
```

Comment:
Nice 👍 Definitely gets tiring writing over and over and I don't think we've been entirely consistent so this would help as a shared utility function